### ISSUE TITLE
Download Link not Working.

### ISSUE THREAD

**Issue by FrozenCoder1** [LewisTian/NeteaseLyric#1]
I've tried using VPN for 28 Different countries but not even a single country is working. 
Please upload files using Google Drive or Mediafire...

**Comment by 18aKissD** [comment]
Yes download link not working, or if it's working, you have to download 3 different programs to download an exe file
